airflow/providers/cncf/kubernetes/triggers/pod.py

```diff
@@ -182,7 +184,8 @@ async def run(self) -> AsyncIterator[TriggerEvent]:  # type: ignore[override]
                                 }
                             )
                             return
-
+                        else:
+                            await asyncio.sleep(self.startup_check_interval)
                     self.log.info("Sleeping for %s seconds.", self.poll_interval)
                     await asyncio.sleep(self.poll_interval)
```

Comment:
We need to add an else to avoid sleeping twice (`startup_check_interval` and `poll_interval`)
```suggestion
                            self.log.info("Sleeping for %s seconds.", self.startup_check_interval)
                            await asyncio.sleep(self.startup_check_interval)
                    else:
                        self.log.info("Sleeping for %s seconds.", self.poll_interval)
                        await asyncio.sleep(self.poll_interval)
```